adam/batch.py

```diff
@@ -135,11 +136,6 @@ def __init__(self, params):
             object_name (str): name of object (default: 'dummy')
             object_id (str): identification of object (default: '001')
 
-            center_name (str): center for propagation. 'SUN' or 'EARTH'. (default: 'SUN')
-            ref_frame (str): reference frame for propagation. 'ICRF' (International Celestial
-                             Reference Frame) or 'EMEME2000' (Earth Mean Ecliptic Mean
-                             Equinox of J2000). (default: 'ICRF')
-
```

Comment:
looks like these are still referenced in `ReferenceFrameTest`

Reply:
Oof, good catch. This clobbered a bunch of fixes that I'd made in a different pull request. Putting them back now...